services/spd/tlkd/tlkd_main.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, ARM Limited and Contributors. All rights reserved.
+ * Copyright (c) 2015-2016, ARM Limited and Contributors. All rights reserved.
```

Comment:
This patch was created in 2016. I am just pushing it upstream. Any specific reason to update it to 2017?